packages/shared/src/utils/transactions.ts

```diff
@@ -92,7 +92,7 @@ export const getTransactionInfoType = (
       return 'swap'
     } else {
       const alphAmountReduced = alphAmount < 0 // tokenAmounts is checked in the swap condition
-      const isInternalTransfer = isInternalTx(tx, internalAddresses)
+      const isInternalTransfer = isWalletSelfTransfer(tx, internalAddresses)
```

Comment:
this func will eventually become obsolete so I wouldn't bother renaming things in it for now.